src/tespy/components/turbomachinery/steam_turbine.py

```diff
@@ -0,0 +1,289 @@
+            num_eq=1, elements=["alpha", "eta_s_dry"],
+            func=self.eta_s_wet_func,
+            deriv=self.eta_s_wet_deriv,
+            latex=self.eta_s_wet_func_doc)
+
+        return params
+
+    def eta_s_wet_func(self):
+        r"""
+                Equation for given dry isentropic efficiency of a turbine under wet expansion.
+
+                Returns
+                -------
+                residual : float
+                    Residual value of equation.
+
+                    .. math::
+
+                        0 = -\left( h_{out} - h_{in} \right) +
+                        \left( h_{out,s} - h_{in} \right) \cdot \eta_{s,e}
+
+                        \eta_{s,e} = \eta_{s,e}^{dry} \cdot \left( 1 - \alpha \cdot y_m \right)
+
+                        y_m = \frac{\left( 1-x_{in}\right)+ \left( 1-x_{out} \right)}{2}
+                """
+
+        inl = self.inl[0]
+        outl = self.outl[0]
+
+        state = inl.calc_phase()
+        if state == "tp":  # two-phase or saturated vapour
+
+            ym = 1 - (inl.calc_x() + outl.calc_x()) / 2  # average wetness
+            return self.calc_eta_s() - self.eta_s_dry.val * (1 - self.alpha.val * ym)
+
+        else:  # superheated vapour
+            dp = inl.p.val_SI - outl.p.val_SI
+
+            # compute the pressure and enthalpy at which the expansion
+            # enters the vapour dome
+            def find_sat(frac):
+                psat = inl.p.val_SI - frac * dp
+
+                # calculate enthalpy under dry expansion to psat
+                hout_isen = isentropic(
+                                    inl.p.val_SI,
+                                    inl.h.val_SI,
+                                    psat,
+                                    inl.fluid_data,
+                                    inl.mixing_rule,
+                                    T0=inl.T.val_SI
+                                )
+                hout = inl.h.val_SI - self.eta_s_dry.val * (inl.h.val_SI - hout_isen)
+
+                # calculate enthalpy of saturated vapour at psat
+                hsat = h_mix_pQ(psat, 1, inl.fluid_data)
+
+                return hout - hsat
+
+            frac = brentq(find_sat, 1, 0)
+            psat = inl.p.val_SI - frac * dp
+            hsat = h_mix_pQ(psat, 1, inl.fluid_data)
+
+            # calculate the isentropic efficiency for wet expansion
+            ym = 1 - (1.0 + outl.calc_x()) / 2  # average wetness
+            eta_s = self.eta_s_dry.val * (1 - self.alpha.val * ym)
+
+            # calculate the final outlet enthalpy
+            hout_isen = isentropic(
+                psat,
+                hsat,
+                outl.p.val_SI,
+                inl.fluid_data,
+                inl.mixing_rule,
+                T0=inl.T.val_SI
+            )
+            hout = hsat - eta_s * (hsat - hout_isen)
+
+            # calculate the difference in enthalpy
+            dh_bisectioned = hout - inl.h.val_SI
```

Comment:
I changed this from `dh_isen` to `dh_bisectioned` (we can also make it just sectioned or anything better you may com up with). The reason for this is, that `dh_isen` made me think of the isentropic delta h, but actually it is the real delta h, when factoring in the dry and wet expansion sections, or am I mistaken?